crates/nargo_cli/tests/test_data_ssa_refactor/simple_body_array_return/src/main.nr

```diff
@@ -0,0 +1,5 @@
+// Simple program to test the returning of arrays
+// declared withing the functino body.
+fn main(x : Field) -> pub [Field; 2] {
+    [x, x]
+}
```

Comment:
Can we make this a unit test rather than an integration test?

In general I'd rather small, localized tests like this or regression tests for past bugs be unit tests so that we don't have to run the full frontend machinery on each test.

Reply:
Done